src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -165,8 +169,12 @@ launchNotifier nat =
         , updateNotifier
         ]
   where
-    cooldownPeriod = 5000000         -- 5 sec
+    cooldownPeriod :: Microsecond
+    cooldownPeriod = 5000000
```

Comment:
Ok, I agree. This commit fixes this: https://github.com/input-output-hk/cardano-sl/pull/85/commits/95b79b7ef215593a1c2035a6bf8f93fb9d408569